app/src/main/java/com/duycuong/weather/ui/screen/main/nextdays_weather/NextDaysViewModel.java

```diff
@@ -0,0 +1,66 @@
+package com.duycuong.weather.ui.screen.main.nextdays_weather;
+
+import android.content.Context;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.location.Location;
+import android.location.LocationManager;

```

Comment:
ERROR: Unused import: android.location.LocationManager. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports